lib/update.ml

```diff
@@ -993,6 +1000,28 @@ let reconstitute_date_dmy conf var =
   in
   (d, force_f_cal)
 
+let is_illegal_access_update old_access new_access =
+  match old_access, new_access with
+  | IfTitles, Public
+  | IfTitles, Private
+  | Public, Private
+  | Private, Public -> true
+  | _ -> false
```

Comment:
Could you please enumerate all the cases to make this pattern matching future-proof?